src/Program/ProgramRequest.php

```diff
@@ -51,15 +66,20 @@ public function getResponseObject(): AbstractEntity
 
     public function jsonSerialize()
     {
-        return [
+        $data = [
             "name" => $this->name,
             "point" => $this->point,
             "phone" => $this->phone,
             "url" =>$this->url,
             "active" =>$this->active,
             "logo" => $this->logo,
             "organization" => $this->organization,
-
         ];
+        if ($this->getEnableEmailLogin() !== null) {
```

Comment:
if you had the boolean, like zech mentioned.     if ($this->isEmailLoginEnabled()) { it sure is }

Reply:
this is for the PUT request setter